doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -420,6 +420,11 @@ Trusted Firmware-M
 
 * Fixed the NCSDK-14015 known issue that would cause crash during boot when the :kconfig:`CONFIG_RPMSG_SERVICE` Kconfig option was enabled on the nRF5340 SoC.
 
+cJSON
+=====
+
+*  Fixed an issue with floats in the cJSON module when using NEWLIB_LIBC without the :kconfig:`NEWLIB_LIBC_FLOAT_PRINTF` Kconfig option.
```

Comment:
Shouldn't this now be `:kconfig:option:`? CC @divipillai

Reply:
yes

Reply:
Thanks, updated it with the rebase :)